PR from V2 branch - added  section for Cloud Observability.

stephenmogg opened the pull request.

LukaszMarchewka committed:
added link from README
Added detail on opentelemetry
Added new section for Cloud Observability.

LouisParkin commented on suse-cloud-observability-quick-start-guide.md:
```diff
@@ -0,0 +1,372 @@
+
+### Supported versions
+
+| Supported Kubernetes Version |
+|------------------------------|
+| Kubernetes 1.30              |
+| Kubernetes 1.29              |
+| Kubernetes 1.28              |
```
I updated the main one recently, added support up to Kubernetes 1.32.

LouisParkin replied:
See here -> https://docs.stackstate.com/get-started/k8s-quick-start-guide#supported-versions

LouisParkin commented on suse-cloud-observability-quick-start-guide.md:
```diff
@@ -0,0 +1,372 @@
+
+| Supported runtime   |
+|---------------------|
+| Docker              |
```
```suggestion
```

LouisParkin commented on suse-cloud-observability-quick-start-guide.md:
```diff
@@ -0,0 +1,372 @@
+[comment]: <> (https://access.redhat.com/support/policy/updates/openshift)
+
+| OpenShift Version | Supported Kubernetes Version | OpenShift End of Support |
+|-------------------|------------------------------|--------------------------|
+| OpenShift 4.12    | Kubernetes 1.25              | July 17, 2024            |
+| OpenShift 4.11    | Kubernetes 1.24              | February 10, 2024        |
+| OpenShift 4.10    | Kubernetes 1.23              | September 10, 2023       |
```
See here https://docs.stackstate.com/get-started/k8s-quick-start-guide#openshift

LouisParkin commented on suse-cloud-observability-quick-start-guide.md:
```diff
@@ -0,0 +1,372 @@
+
+| Supported runtime   |
+|---------------------|
+| Docker              |
```
```suggestion
```

LouisParkin commented on suse-cloud-observability-quick-start-guide.md:
```diff
@@ -0,0 +1,372 @@
+
+| Kubernetes version | Amazon EKS release | Amazon EKS End of Support | Amazon EKS End of Extended Support |
+|--------------------|--------------------|---------------------------|------------------------------------|
+| 1.30               | May 23, 2024       | July 23, 2025             | July 23, 2026                      |
+| 1.29               | January 23, 2024   | March 23, 2025            | March 23, 2026                     |
+| 1.28               | September 26, 2023 | November 01, 2024         | November 26, 2025                  |
+| 1.27               | May 24, 2023       | July 2024                 | July 24, 2025                      |
```
See here -> https://docs.stackstate.com/get-started/k8s-quick-start-guide#amazon-eks

LouisParkin commented on suse-cloud-observability-quick-start-guide.md:
```diff
@@ -0,0 +1,372 @@
+
+| Supported runtime  |
+|--------------------|
+| Docker             |
```
```suggestion
```

LouisParkin reviewed (changes requested):
The changes I made, I made for all the k8s flavours, just match your content with https://docs.stackstate.com/get-started/k8s-quick-start-guide

Or look at commit 9236bb1a2be7253fadabb8b50b7bdec6847ffd93

stephenmogg committed:
Update suse-cloud-observability-quick-start-guide.md

stephenmogg committed:
Update suse-cloud-observability-quick-start-guide.md

Removed supported Kubernetes version info and pointed to quick start guide - rather than maintain 2 lists.

stephenmogg commented:
I removed the version information and put a link to the quickstart page  (Rather than maintain 2 lists)

LouisParkin reviewed (approved):
Looks good to me.
